src/coreComponents/physicsSolvers/wavePropagation/AcousticVTIWaveEquationSEMKernel.hpp

```diff
@@ -0,0 +1,576 @@
+   *
+   */
+  GEOS_HOST_DEVICE
+  GEOS_FORCE_INLINE
+  void quadraturePointKernel( localIndex const k,
+                              localIndex const q,
+                              StackVariables & stack ) const
+  {
+    // Pseudo Stiffness xy 
+    m_finiteElementSpace.template computeStiffnessxyTerm( q, stack.xLocal, [&] ( int i, int j, real64 val )
+    {
+      real32 const localIncrement_p = val*(-1-2*m_epsilon[k])*m_p_n[m_elemsToNodes[k][j]];
+      RAJA::atomicAdd< parallelDeviceAtomic >( &m_stiffnessVector_p[m_elemsToNodes[k][i]], localIncrement_p );
+      real32 const localIncrement_q = val*((-2*m_delta[k]-m_vti_f[k])*m_p_n[m_elemsToNodes[k][j]] +(m_vti_f[k]-1)*m_q_n[m_elemsToNodes[k][j]]);
+      RAJA::atomicAdd< parallelDeviceAtomic >( &m_stiffnessVector_q[m_elemsToNodes[k][i]], localIncrement_q );
+
+      //DEbug
+      real32 const localIncrement = val*m_p_n[m_elemsToNodes[k][j]];
+      RAJA::atomicAdd< parallelDeviceAtomic >( &m_stiffnessVector[m_elemsToNodes[k][i]], localIncrement );
+    } );
+
+    // Pseudo-Stiffness z
+    
+    m_finiteElementSpace.template computeStiffnesszTerm( q, stack.xLocal, [&] ( int i, int j, real64 val )
+    {
+      real32 const localIncrement_p = val*((m_vti_f[k]-1)*m_p_n[m_elemsToNodes[k][j]] - m_vti_f[k]*m_q_n[m_elemsToNodes[k][j]]);
+      RAJA::atomicAdd< parallelDeviceAtomic >( &m_stiffnessVector_p[m_elemsToNodes[k][i]], localIncrement_p );
+
+      real32 const localIncrement_q = -val*m_q_n[m_elemsToNodes[k][j]];
+      RAJA::atomicAdd< parallelDeviceAtomic >( &m_stiffnessVector_q[m_elemsToNodes[k][i]], localIncrement_q );
+
+      //DEbug
+      real32 const localIncrement = val*m_p_n[m_elemsToNodes[k][j]];
+      RAJA::atomicAdd< parallelDeviceAtomic >( &m_stiffnessVector[m_elemsToNodes[k][i]], localIncrement );
+    } );
+  }
+
+protected:
+  /// The array containing the nodal position array.
+  arrayView2d< real64 const, nodes::REFERENCE_POSITION_USD > const m_X;
+
+  /// The array containing the nodal pressure array.
+  arrayView1d< real32 const > const m_p_n;
+
+  /// The array containing the nodal auxiliary variable array.
+  arrayView1d< real32 const > const m_q_n;
+
+  /// The array containing the product of the stiffness matrix and the nodal pressure for the equation in p.
+  arrayView1d< real32 > const m_stiffnessVector_p;
+
+  /// The array containing the product of the stiffness matrix and the nodal pressure for the equation in q.
+  arrayView1d< real32 > const m_stiffnessVector_q;
+
+  /// DEBUG
+  arrayView1d< real32 > const m_stiffnessVector;
+
+  /// The array containing the epsilon Thomsen parameter.
+  arrayView1d< real32 const > const m_epsilon;
+
+  /// The array containing the delta Thomsen parameter.
+  arrayView1d< real32 const > const m_delta;
+
+  /// The array containing the f parameter.
+  arrayView1d< real32 const > const m_vti_f;
+
+  /// The time increment for this time integration step.
+  real64 const m_dt;
+
+
+};
+
+
```

Comment:
Careful with names, maybe the factory should be named  the same way as the solver itself not using the same name as the isotropic acoustic solver

Reply:
Yes, I changed it to 

```
using ExplicitAcousticVTISEMFactory = finiteElement::KernelFactory< ExplicitAcousticSEM,
                                                                 real64 >;
```